mobile/src/main/java/butter/droid/fragments/ShowDetailFragment.java

```diff
@@ -88,7 +84,7 @@ public void onCreate(Bundle savedInstanceState) {
     public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
         mRoot = inflater.inflate(R.layout.fragment_showdetail, container, false);
         ButterKnife.bind(this, mRoot);
-        if (VersionUtils.isJellyBean() && container != null) {
+        if (container != null) {
```

Comment:
`isJellyBean()` method checks if version code is `16` or higher which is now true every time. So the check is redundant and that it why it was removed.